src/routes/sosContactRoutes.js

```diff
@@ -55,13 +55,36 @@ router
       (err) => {
         if (!err) {
           res.send('Successfully added contact');
+          console.log(req.body);
         } else {
           res.send(err);
         }
       },
     );
   });
 
+router.delete(`/users/:username/delete`, (req, res) => {
+  Contact.updateOne(
```

Comment:
not clear from route what u going to delete, seems like user, but it is contact? so when i create delete user option, whould be overlapping 
there is good explanation in the first part of the article:  https://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#restful

Reply:
changed to '/users/:username/contacts' for now